config/filter.d/mongodb-auth.conf

```diff
@@ -0,0 +1,28 @@
+# Fail2Ban filter for unsuccesfull MongoDB authentication attempts
+#
+#
+
+[Init]
+maxlines = 10
+
+[Definition]
+failregex = ^\s+\[conn(?P<__connid>\d+)\] Failed to authenticate [^\n]*<SKIPLINES>\s+\[conn(?P=__connid)\] end connection <HOST>
```

Comment:
our regex for common usage are normally more stricter as regex I had made for you (anchoring, no catch-all if possible, etc.)